adg/pipelines/scripts/coverage-report.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+if [ $(ls reports/coverage*.xml | wc -l) != "7" ]; then
```

Comment:
Instead of using 7 directly, let's use a variable name for it that describes the intent.

Reply:
Please ignore this for now, This file is not being used, This will also be updated under a new ticket.